esp/applications/eclqueries/ldap_authorization_map.yaml

```diff
@@ -1,4 +1,5 @@
 ldap:
+   resourcesBasedn: ou=WsEcl,ou=EspServices,ou=ecl
```

Comment:
I wonder if 
`resourcesBasedn: ou=WsEcl,ou=EspServices,ou=ecl`
should be removed from esp/applications/common/ldap/ldap.yaml?